hazelcast-client/src/test/java/com/hazelcast/client/cache/ClientCacheIteratorTest.java

```diff
@@ -30,7 +30,7 @@
 import javax.cache.spi.CachingProvider;
 
 
-@RunWith(HazelcastParallelClassRunner.class)
+@RunWith(HazelcastTestRunner.class)
```

Comment:
`@RunParallel` annotation should be added to run parameterized tests in parallel.